omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -389,6 +389,7 @@
 # we'll be using these directories to determine what versions we have installed and
 # whether we need to run pg_upgrade.
 default['private_chef']['postgresql']['enable'] = true
+default['private_chef']['postgresql']['remote'] = false
```

Comment:
How about checking for postgresql['enable'] == false to avoid the false negative. That way, if postgresql['vip'] evaluates to a remote machine, or if it evaluates to a local address, but postgres is disabled, then treat it as remote.